type_check_failed,error_msgs
False,
True,"File ""dummpy_input_file"", line 16, in load: No attribute 'mPatchSize' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_response_for_url: Name 'r' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in update_url: No attribute 'session' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'token' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in getFilledBBPoints: unsupported operand type(s) for item retrieval: 'pointDim: int' and '0: int' [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in getBB: Name 'ymin' is not defined [name-error]"
False,
True,"invalid syntax (<unknown>, line 3)"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in trimList: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __pixelVelocity: Name 'dx' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in evaluate_hand: Name 'straight' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in replace_username: Name 'username' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in post: Name 'form' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in post_image: Name 'image_upload' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in mentions: Name 'user' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in search: No attribute 'user' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in invert: No attribute 'pt2' on list [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in variance: Name 'num' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in std: No attribute '__iter__' on float [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 17, in median: Name 'retVal' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in findFirstIdxEqualTo: unsupported operand type(s) for item retrieval: 'retVal: None' and '0: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 29, in __init__: unsupported operand type(s) for item retrieval: 'numpoints: int' and '1: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in __init__: Name 'at_x' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 3)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _tasklines_from_tasks: unsupported operand type(s) for item retrieval: List[nothing] and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 7, in _prefixes: unsupported operand type(s) for +: 'ps: Dict[nothing, nothing]' and '1: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in edit_task: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 4, in finish_task: Name 'task' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in write: Name 'tasks' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in _build_parser: No attribute 'add_option_group' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in upload: No attribute 'name' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_camera_image: Name 'img' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_eline: Name 'StereoImg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_projectPoint: Name 'img1' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_loadCalibration: Name 'calbib' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_getImagesUndistort: Name 'rectLeft' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_camera_video: Name 'img' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in horizontalTilt: Name 'distance_ratio' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in bootstrap_list: Name 'executing' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in find: Name 'item' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in getFaceSet: Name 'img' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in find_gopaths: No attribute 'filename' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in save_training_history: Name 'train_hist' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in step: No attribute 'reset_episode' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _AddMessageDescriptor: Name 'nested' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in clear_all_keys: Name 'filename' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 36, in start_xmpp_client: No attribute 'add_event_handler' on logging.Logger [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in entered_file_action: No attribute '_create_new_file' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in single_file_action: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in run: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_file_number: Name 'fname' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in toPoints: Name 'bl' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in rebase: unsupported operand type(s) for item retrieval: 'len: Callable[[Sized], int]' and 'x: Any' [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in meanColor: Name 'w' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get: Name 'meta' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in calc_all_hot_window_scores: Name 'stories' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_hot_stories: Name 'stories' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in send_mail: Name 'body' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in hipchat_is_up: Name 'last_status' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in say_bitly_short_url: Name 'response' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in what_time_is_it_in: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _rpcs_send: Name 'status' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in rpcs: No attribute '_notifier' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in flush: Name 'id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in _fetch_remote_file: No attribute 'geturl' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _run: Name 'hostname' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_wait_store: Name 'queue' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in __init__: Can't assign attribute 'bot_plugins' on str [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in autoping: No attribute 'last_ping' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in output: No attribute 'do_output' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in do_output: No attribute 'module' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _extractFromBinary: Name 'temp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _getHoles: No attribute 'h_next' on List[list] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in imgIntersections: No attribute 'end_points' on List[Tuple[Any, int]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in crop: No attribute 'window' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in has_trace_limits: Name 'r' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in __init__: Can't assign attribute '_des_func' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in add_place: unsupported operand type(s) for item retrieval: Dict[str, Any] and Type[str] [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in downgrade: Name 'comments' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in downgrade: Name 'users' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in user_preferences: No attribute 'POST' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in clear: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in startService: Name '_kwargs' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in startService: Name '_options' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_load_all: Name 'queue' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in input_panel_caption: No attribute 'is_python' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in update_status_message: No attribute 'view' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in generate_project_auto_complete: No attribute 'generate_auto_complete' on list [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in generate_auto_complete: Name 'entry' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in main: Name 'test_input' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 39, in run: Can't assign attribute 'duration_ms' on int [not-writable]"
False,
True,"File ""dummpy_input_file"", line 21, in __repr__: No attribute 'inputs' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in pass_test: No attribute 'test' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in add_output: Name 'test_out' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _process_user_options: Name 'k' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _write_wsconf: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in delete: Name '_parent' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 'verbose' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in assertIn: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_file_upload: Function os.close was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_blank_access_token: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_clear: No attribute 'deque' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in startService: Name '_options' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in enhanced_status_code: Name 'code_0' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in twit_sign: Name 'twit_auth' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in twitter_account_for_user: Name 'twitter_user' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in twitter_verify_credentials: Name 'json' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_command_class_from_apps: Name 'command_class' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __exit__: No attribute 'depth' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in usage: No attribute 'help' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in str_with_codes: No attribute 'code' on tuple [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in shell: Built-in function input was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __getattr__: Name 'light' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_list_subnets_with_filters: No attribute 'assert_called_with' on Dict[str, Dict[str, Dict[str, Union[Dict[str, str], int, str]]]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_resolve_subnet_ids: Name '_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_image: No attribute 'vgbdtg' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in auth: Name 'headers' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in handle_error: No attribute 'request' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 6)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 5)"
False,
False,
False,
True,"invalid syntax (<unknown>, line 3)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 2)"
False,
True,"invalid syntax (<unknown>, line 2)"
False,
True,"invalid syntax (<unknown>, line 5)"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in copy_sample: Name 'model_copy' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in log_likelihood: Name 'ind' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in redo: Name 'row' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in link_data_changed: Name 'matches' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in add_files: No attribute 'gui' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_list_keys: No attribute 'assert_called_with' on Dict[str, Dict[str, Dict[str, str]]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_resolve_ids_label: Name '_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_create_ticket: No attribute 'assert_called_with' on Tuple[Dict[str, Union[int, str]], str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in setPauseOnExceptions: Name 'command' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in setSkipAllPauses: Name 'command' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_create_snapshot_space: Name 'mock' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_ip_lookup: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in resolve_ids: Name 'ids' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_vlb: No attribute 'data_list' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in compute_rate_at_events: Name 'C' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in create_connection: No attribute 'connect' on Tuple[()] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_line_mapping_count: Name 'content' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in registerAndRunTests: Name 'suite' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in list_accounts: No attribute 'getObject' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_origins: No attribute 'account' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in list_zones: No attribute 'client' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_add_subnet_for_ipv6: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _record_call: No attribute 'append' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in setUp: Name 'wrapped_transport' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in call_has_props: Name 'actual_value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in run_forever: Name 'data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in _run_with_no_err: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_generated_position_casing: Name 'position' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_create_invalid_mapping: Name 'mapping' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in create_task: Name 'task_val' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in aes_encrypt: No attribute 'encrypt' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_build_client: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_list_accounts: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_load_content: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_list_origins: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_zone_print: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_list_zones: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_add_record: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_import_zone_dry_run: No attribute 'assertIn' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_import_zone: No attribute 'calls' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_add_by_option: Name 'service' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in log_probability: Name 'lp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in expected_W: Name 'E_W' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 3)"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in cli: Name 'mq_client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in push_queue_message: No attribute 'json' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in pop_message: Name 'messages' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_topic: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_params: Name 'params' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in docs: No attribute 'getElementsByTagName' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in texts: Name 'doc' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in docs: No attribute '_csv_file' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_command: No attribute 'commands' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in parse_rules: unsupported operand type(s) for item retrieval: 'order: int' and '0: int' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in cli: Name 'mgr' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 22, in cli: Name 'rate' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in cli: No attribute 'get' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in register_object: Name 'attribute' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in is_sijax_request: Name 'k' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in process_request: No attribute 'execute_callback' on dict [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in alert: No attribute '_add_command' on dict [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in _html: No attribute 'COMMAND_HTML' on Type[dict] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in script: Name 'params' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in lines: No attribute 'view' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in rows: No attribute 'view' on list [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in remove_click: Name 'regions' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in check_click: Name 'cursor' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in wrap_view: Name 'id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in format: No attribute 'rsv3' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in mask: Name 'i' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in sstpControlPacketReceived: Name 'nonce' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 38, in sstpMsgCallConnectRequestReceived: Name 'nak' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in close: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_complex: unsupported operand type(s) for item retrieval: Dict[str, Dict[str, Dict[str, str]]] and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_add_key: No attribute 'assert_called_with' on Tuple[Dict[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in unregister: No attribute '_ptr' on str [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in incoming_midi_events: Name 'err' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in from_notebook_node: No attribute 'cells' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 6)"
False,
False,
False,
False,
False,
False,
False,
True,"invalid syntax (<unknown>, line 11)"
False,
True,"File ""dummpy_input_file"", line 4, in _moving_average: Name 'ret' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in dataframe_to_mat: No attribute 'location' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 14, in run: Name 'mat' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in _handle_network: unsupported operand type(s) for item retrieval: 'private_network_only: bool' and '0: int' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 8, in get_list_params: No attribute 'get_param' on Dict[str, Dict[str, Dict[str, Union[List[Dict[str, Union[List[str], str]]], str]]]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in sstpMsgEchoRequest: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_audio_mode: unsupported operand type(s) for item retrieval: List[str] and TextIO [unsupported-operands]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in blit_time: No attribute 'clock_text' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in remove: No attribute '_add_command' on dict [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in has_schema: Name 'basename' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __repr__: Name 'strings' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __new__: Name 'schema_attrs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_boolean: No attribute 'assertCompliant' on List[Union[bool, int]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in GuessLanguage: Name 'extension' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in IPythonShell: Name 'ipshell' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in FindFileInDirectoryOrAncestors: Name 'filepath' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in GetDBPathRelativeToCurrentDirectory: Name 'abs_filepath' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in ConvertTree: Name 'visitor' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in EscapeWhitespace: Name 'a' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'encoding' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 3, in stringToBytes: Name 'bytes' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in stringToNumber: Name 'bytes' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _addPKCS1SHA1Prefix: Name 'prefixedBytes' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in ConvertTree: Name 'ast' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in TestAllPrograms: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in TestNiceInitKeyValueStore: Name 'store' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in setUp: Name 'store' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in CreateNewStore: Name 'store' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in IsValidStore: Name 'store' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in ping: Name 'resp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in _make_request: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in stats: Name 'resp' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in create_queue: No attribute 'json' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in cli: Name 'key_text' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in cli: No attribute 'add_row' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in _parse_create_args: Name 'userfile' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _validate_args: unsupported operand type(s) for item assignment: list and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in call: No attribute 'COMMAND_CALL' on Type[list] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in _process_callback: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_registering_custom_events_works: No attribute 'register_event' on Callable[[Any], Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_wrapped_methods: Name 'wrapped_methods' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_empty_namenodes_haclient: No attribute 'assertRaises' on Tuple[()] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in usage: Name 'sub_cmd' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'targets' on tuple [not-writable]"
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'i' on Dict[str, Any] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'edge_cov' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in build: Name 'st' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_remove: Name 'lt' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_date: No attribute 'assertCompliant' on List[Union[float, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __repr__: Name 'strings' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in from_json_compatible: No attribute 'load' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in create: Name 'node' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in run: Name 'node' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in rawlatex2math: Name 'json_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _get_empty_lines: Name 'before' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _set_empty_lines: Name 'value' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in visit_admonition_html: Name 'classes' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in handle500: Name 'errorid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __CreateGraphNode: Name 'node_index' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in AddASTNodeToGraph: Name 'node' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __ConvertToElement: No attribute '__AddChildToElement' on dict [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __ParseLAST: Name 'parser' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in __ParseLanguageSpecificTree: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in callback_before_new: No attribute 'alert' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in determine_version: No attribute 'search' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __get_all_directories: No attribute 'args' on List[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_copyToLocal_conflicting_names: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in runTests: Name 'error_count' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_apps: Name 'splunk_apps' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_current_app: Name 'app_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in handle404: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_changelist_disallows_password_lookups: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in render_options: No attribute '_widgets' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in widget: Name 'context' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in vendor: No attribute 'dashboard' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 31, in lookup_field: Name 'sub_rel_name' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_app_label: Name 'ret' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get: No attribute 'user' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in request: Name 'current_line' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in _get_projects_from_session: No attribute 'get' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_register_model: Name 'site' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in save: Name 'fp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in do_POST: No attribute 'rfile' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in format: No attribute 'levelname' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in format_completion: Name 'display' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _ContractAperture: No attribute '_pending_endpoints' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in _Get: Name 'sink' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _OpenImpl: Name 'sink' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __call__: No attribute '_fn' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute 'last_update' on List[nothing] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in RecordPercentileSample: Name 'reservoir' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in readAll: unsupported operand type(s) for +: int and memoryview [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 7, in Sample: No attribute '_last' on float [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in _insert_characters: No attribute 'end' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_plugin_settings: Name 'plugin_settings' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in call_callback: Name 'callback' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _check_login_social_session: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_login_social_http_origin: Name 'resp' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_login_manual_redirect: Name 'resp' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_driver: Name 'provider_class' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in update_config: Name 'profile_yaml' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in validate: Name 'attr' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in update: Name 'settings' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in save: No attribute 'errors' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_unreadable_settings_file: No attribute 'assertRaises' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in render_in_place: Name 'f' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in setUp: No attribute 'log' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in setup: Name 'req' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in get_init_widget: Name 'widget' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in get_context: Name 'new_context' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 3, in get: No attribute 'template_response' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_context: Name 'new_context' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in init_request: No attribute 'prepare_form' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in get: No attribute 'instance_forms' on tuple [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in label: Name 'text' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in init_request: Can't assign attribute 'to_field' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in merge_class: Name 'meta_class' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_list_display: Name 'list_display' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in setup: Name 'req' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in init_request: No attribute 'get_model_view' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in init_request: No attribute 'org_obj' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_artifact_group: Name 'parts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in resolve_artifact: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in file_link: No attribute 'filename' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in queryset: Name 'queryset' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __init__: Can't assign attribute 'name' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in __init__: No attribute 'initial_wait_interval' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in register_models: Name 'admin' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 2, in init_request: No attribute 'reversion_enable' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in init_request: No attribute 'has_change_permission' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_form_datas: No attribute 'request_method' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_context: Name 'context' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in init_request: No attribute 'reversion_enable' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in _method: Name 'comment' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_context: Name 'new_context' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in block_pagination: Name 'pagination_required' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in do_rest_login: Name 'start_url' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in do_rest_login: Name 'start_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in save_model: Name 'email' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_queryset: Name 'qs' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in init_request: No attribute 'request' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in init_request: No attribute 'enable_themes' on tuple [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in _register_model: Name 'ct_fk_field' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in set_artifact_metadata: Name 'value' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in create_staging_repo: Name 'profile' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _get_target_repository: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_create_list_and_drop: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in setUp: No attribute '_waitForSink' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in sendLookAt: Name 'msg' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in sendLookFrom: Name 'dest' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in unpack_state: Name 'q' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_update_env: Name 'gconf' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 2, in __exit__: No attribute 'stop' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in test_isinstance_works: No attribute 'stop' on datetime.date [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 44, in update_gunicorns: No attribute 'index' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in create_cookie_data: Name 'cookie_data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_cookie_hash: Function hmac.new was called with the wrong arguments [wrong-arg-types]"
False,
True,"File ""dummpy_input_file"", line 4, in test_form_post_success: Name 'wizard' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_template: No attribute 'assertTemplateUsed' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in __init__: Can't assign attribute 'defaults' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in formfield_for_dbfield: No attribute 'widget' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_view_class: No attribute 'admin_site' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in make_get_request: Name 'req' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in context: No attribute '_meta' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in _inject_rules: No attribute 'get_security_groups' on Dict[Any, list] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in rules: Name 'groups' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in validate_specfile: No attribute 'get' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in all: No attribute 'list' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in auto_paging_iter: No attribute 'list' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 35, in instance_url: Name 'account' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in mark_as_fraudulent: No attribute 'refresh_from' on Dict[str, Dict[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in invoice_items: No attribute 'id' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in delete_discount: Name 'url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in install_vexts: Name 'dest' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_urls: No attribute 'resources' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in save: Can't assign attribute 'parent_id' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in __unicode__: Name 'term' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in module_feedback: Name 'feedback' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in delete_models: No attribute 'get_objects' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in new_generic: Name 'id' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Name 'obj' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
